src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -4,24 +4,41 @@
 # Author :  Andrii Lytovchenko
 ########################################################################
 """
-Login to DIRAC.
+With this command you can log in to DIRAC.
+
+There are two options:
+
+  - using a user certificate, creating a proxy.
+  - go through DIRAC Authorization Server by selecting your Identity Provider.
+
 
 Example:
-  $ dirac-login -g dirac_user
+  # Login with default group
+  $ dirac-login
+  # Choose another group
+  $ dirac-login dirac_user
+  # Return token
+  $ dirac-login dirac_user --token
 """
 from __future__ import division
 from __future__ import absolute_import
 from __future__ import print_function
 
```

Comment:
You can remove these imports now that it is all python3

Reply:
done